js/shared.js

```diff
@@ -1256,6 +1257,93 @@ export function openNewTab(url) {
   }
 }
 
+export const usGov = `
+You are accessing a U.S. Government web site which may contain information that must be protected under the U.S. Privacy Act or other sensitive information and is intended for Government authorized use only. Unauthorized attempts to upload information, change information, or use of this web site may result in disciplinary action, civil, and/or criminal penalties. Unauthorized users of this web site should have no expectation of privacy regarding any communications or data processed by this web site. Anyone accessing this web site expressly consents to monitoring of their actions and all communication or data transitioning or stored on or related to this web site and is advised that if such monitoring reveals possible evidence of criminal activity, NIH may provide that evidence to law enforcement officials.
+`;
+
+export const firebaseSignInRender = async ({ ui, account = {}, usGov}) => {
```

Comment:
sorry to be picky on this - but I don't think `usGov` should be a parameter for this function. Just set it as a variable in this function using the shared value from this file. Then we don't need to import it and pass it from the other pages.

Reply:
I found it's also used in setting.js

https://github.com/episphere/connectApp/blob/d56020fd3b6779460480e131e5877cafbc32c11c/js/pages/settings.js#L459
https://github.com/episphere/connectApp/blob/d56020fd3b6779460480e131e5877cafbc32c11c/js/pages/settings.js#L485

So i think it makes sense to pass the usGov into function

Reply:
Two questions, why in this scenario are we passing an empty string for `usGov` ? If it's not supposed to be empty, I'm assuming it's supposed to be the same string that we use elsewhere in the code. If that's the case, then usGov is never really change as an input parameter so just reference it inside the function `firebaseSignInRender`.

This all depends on the answer to the first question.

Reply:
Two questions, why in this scenario are we passing an empty string for usGov ? If it's not supposed to be empty, I'm assuming it's supposed to be the same string that we use elsewhere in the code. If that's the case, then usGov is never really change as an input parameter so just reference it inside the function firebaseSignInRender.

This all depends on the answer to the first question.

Reply:
I'm not sure if everyone has agreed on this yet, but I noticed the homepage screen have usGov

<img width="294" alt="image" src="https://github.com/episphere/connectApp/assets/24559107/a3ed65c0-3c35-4997-989d-97b123a8d175">


And the setting screen did not have usGov

![image](https://github.com/episphere/connectApp/assets/24559107/41abb339-4033-409e-8600-71ac229be344)